components/Card/ProjectCard.tsx

```diff
@@ -32,47 +44,48 @@ export function ProjectCard(props: { vsm: vsmProject }): JSX.Element {
       return unfaveProject(props.vsm.vsmProjectID);
     },
     {
-      onSettled: () =>
-        queryClient
-          .invalidateQueries()
-          .then(() => setIsMutatingFavourite(false)),
+      onSettled: handleSettled,
     }
   );
 
   return (
-    <Link href={`/process/${props.vsm.vsmProjectID}`}>
-      <div className={styles.card}>
-        <div className={styles.topSection}>
-          <div className={styles.vsmTitleContainer}>
-            <h1 className={styles.vsmTitle}>
-              {props.vsm.name || "Untitled Process"}
-            </h1>
+    <>
+      <Link href={`/process/${props.vsm.vsmProjectID}`}>
+        <div className={styles.card}>
```

Comment:
Could you make this a button so that we support keyboard navigation for going into processes?